database-couchdb/index.js

```diff
@@ -1,103 +1,100 @@
-const fs = require('fs');
+
+const axios = require('axios');
 const PATH = require('path');
-const csvParser = require('csv-parser');
-const Axios = require('axios');
+require('dotenv').config({ path: PATH.join(__dirname, '..', '.env') });
 
-const availabilityPath = PATH.resolve(__dirname, '..', 'data-gen', 'csv', 'availability.csv');
-const storesPath = PATH.resolve(__dirname, '..', 'data-gen', 'csv', 'stores.csv');
 
-const startingItemId = 100;
-const batchSize = 100;
+const { COUCH_URL, COUCH_PWD, COUCH_USER, COUCH_DB } = process.env;
+const url = `http://${COUCH_USER}:${COUCH_PWD}@${COUCH_URL}/${COUCH_DB}`;
+const startId = 100;
 
-const removeOldData = (url, db) => {
+//revision ID helper function needed for couchDB update and delete operations
+//Aka a very specific get request only used to extract the revision id
+const getRevId = (itemId) => {
```

Comment:
I like this helper function! Something that turned me off from couchdb was having to need a revision id to delete?? Seems like an unnecessary extra step, like this function that takes care of that